doc/submitting-changes.xml

```diff
@@ -224,6 +224,139 @@ Additional information.
 </section>
 
 <section>
+  <title>Pull Request Template</title>
+  <para>
+    The pull request template helps determine what steps have been made for a
+    contribution so far, and will help guide maintainers on the status of a
+    change. The motivation section of the PR should include any extra details
+    the title does not address and link any existing issues related to the pull
+    request.
+  </para>
+  <para>When a PR is created, it will be pre-populated with some checkboxes detailed below:
+  </para>
+  <section>
+    <title>Tested using sandboxing</title>
+    <para>
+      When sandbox builds are enabled, Nix will setup an isolated environment
+      for each build process. It is used to remove further hidden dependencies
+      set by the build environment to improve reproducibility. This includes
+      access to the network during the build outside of
+      <literal>fetch*</literal> functions and files outside the Nix store.
+      Depending on the operating system access to other resources are blocked
+      as well (ex. inter process communication is isolated on Linux); see <link
+      xlink:href="https://nixos.org/nix/manual/#description-45">build-use-sandbox</link>
```

Comment:
@other reviewer 
Since both documents are in the same documentation, what is the right way to link them in docbook?

Reply:
I don't think you can because these are in separate documents. This is linking the nix manual, but this is in the nixpkgs manual. Someone that knows docbook better than me can correct me if I'm wrong here.